src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
+
+	if( m_saveIntervalSlider->value() != value )
+	{
+		m_saveIntervalSlider->setValue( value );
```

Comment:
What's the downside to removing this conditional and just setting this each time?